join-metadata-and-clades.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+import argparse
+import re
+import sys
+import pandas as pd
+
+NEXTCLADE_JOIN_COLUMN_NAME = 'seqName'
+VALUE_MISSING_DATA = '?'
+
+column_map = {
+    "clade": "clade",
+    "outbreak": "outbreak",
```

Comment:
I believe "outbreak" is a very specific column for mpox.

I expect that each pathogen can have unique columns in the Nextclade output (e.g. [ncov-ingest includes SC2 specific columns](https://github.com/nextstrain/ncov-ingest/blob/5aed3ed8b8e1bd0a8360f7135f1602df4e01d139/bin/join-metadata-and-clades#L21-L23)). We should probably make the column map customizable to support these. It might be easiest to add an input for a TSV column map that can override the default defined in the script?

Reply:
Added column map customization in 7df16368490e8549a9ebfe2250e6d0e65b716d23
I moved the old columns to default but also open to dropping the defaults altogether.